core/schema/service.go

```diff
@@ -18,9 +18,40 @@ var _ SchemaResolvers = &serviceSchema{}
 
 func (s *serviceSchema) Install() {
 	dagql.Fields[*core.Container]{
-		dagql.Func("asService", s.containerAsService).
+		dagql.Func("asService", s.containerAsServiceLegacy).
+			View(BeforeVersion("v0.15.0")).
 			Doc(`Turn the container into a Service.`,
 				`Be sure to set any exposed ports before this conversion.`),
+
+		dagql.Func("asService", s.containerAsService).
+			View(AfterVersion("v0.15.0")).
+			Doc(`Turn the container into a Service.`,
+				`Be sure to set any exposed ports before this conversion.`).
+			ArgDoc("args",
+				`Command to run instead of the container's default command (e.g., ["run", "main.go"]).`,
```

Comment:
This example assumes there's a `["go"]` entrypoint, right?

Reply:
yes and that it is enabled.

Reply:
We want to discourage the usage of entrypoint, so our examples, docs, even our own code, should avoid it too.

Reply:
E can see `withExec` has the same example, but we should change both.